ast/term.go

```diff
@@ -1102,7 +1102,7 @@ func (ref Ref) Ptr() (string, error) {
 	return strings.Join(parts, "/"), nil
 }
 
-var varRegexp = regexp.MustCompile("^[[:alpha:]_][[:alpha:][:digit:]_]*$")
+var VarRegexp = regexp.MustCompile("^[[:alpha:]_][[:alpha:][:digit:]_]*$")
```

Comment:
I think I'd rather avoid exporting the RE and do something like
```rego
func IsVar(s string) bool {
    return varRegexp.MatchString(s)
}
```
instead. Would it do the trick, here, too?

Reply:
Good point 👍 . I'll fix that.